compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/dnd/ExternalDrag.desktop.kt

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.dnd
```

Comment:
Better to move it to `androidx.compose.ui`

Reply:
What's the difference? What if I want to extract some classes from this file? I am not sure that I want to waste `androidx.compose.ui` Or are any reasons to put it there?

Reply:
We enter into the very hard scientific problem how to name things :).

But if we just look at the other packages in the project (especially in foundation and material), we see that we create individual packages only if there will be a lot of functionality in it, not just one functionaility (no matter how it be decomposed by classes or functions). There are some exceptions, when people created small packages though.

Drag&drop seems a minor feature, and it seems there is no hypothetical functionality that can be put in this package as well in the future?

Reply:
Thank you, got it! Will move it out of dnd package!